android/brave_java_sources.gni

```diff
@@ -9,6 +9,12 @@ brave_java_sources = [
   "../../brave/android/java/org/chromium/chrome/browser/BraveFeatureList.java",
   "../../brave/android/java/org/chromium/chrome/browser/BraveRelaunchUtils.java",
   "../../brave/android/java/org/chromium/chrome/browser/appmenu/BraveAppMenuPropertiesDelegateImpl.java",
+  "../../brave/android/java/org/chromium/chrome/browser/BraveRewardsNativeWorker.java",
```

Comment:
we're starting to get quite a lot of files in this list and will continue to grow. Should we start breaking them up in to `brave_rewards_java_sources`, etc... and then appending them to brave_java_sources?

Reply:
Chromium puts all java sources in one file `chrome_java_sources.jni`. Not sure it will bring some benefits to it, they are in a different folders which are self explaining as to me.

Reply:
that's fine, we can revisit later for general java source organization